src/localization/locales/en-US.json

```
@@ -681,6 +678,10 @@
         "default_marker_label": "Marker",
         "latitude": "latitude",
         "longitude": "longitude",
+        "boundaries_file_empty": "The file {{name}} is empty.",
+        "boundaries_file_invalid_json": "The file {{name}} is a valid JSON file.",
```

```suggestion
        "boundaries_file_invalid_json": "The file {{name}} is a valid JSON file.",
```
question: Is this valid or  not valid? Label says one thing. Text says another.